src/pybind/mgr/orchestrator.py

```diff
@@ -365,24 +366,16 @@ class OsdCreationSpec(object):
     The drive names used here may be ephemeral.
     """
     def __init__(self):
-        self.format = None  # filestore, bluestore
-
-        self.node = None  # name of a node
+        self.host = None  # name of a node
```

Comment:
Please make sure, you're not breaking rook here.

Reply:
I have take care about this and i think it is working as before. Probably we should remove this class in the future.
Any help testing this with the Rook operator will be welcome.